tsconfig.json

```diff
@@ -1,19 +1,11 @@
 {
```

Comment:
Do we still need tsconfig if we're having this as plain js?

Reply:
The`tsconfig.json` is used for running test cases for TypeScript files. I will move it to more appropriate place,`__tests__` dir